test/e2e/specs/plugins/plugins__purchase.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	CartCheckoutPage,
+	TestAccount,
+	RestAPIClient,
+	BrowserManager,
+	SecretsManager,
+	envVariables,
+	PluginsPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Plugins: Add two plugins to the cart' ), function () {
```

Comment:
**suggestion:non-blocking** - this is a more recent development and not recorded anywhere (yet) but we're moving away from the `DataHelper.createSuiteTitle`. You don't have to, but if you'd like you can remove it and just define the title directly like this:

```
describe( 'Plugins: Add two plugins to the cart' , function () {
```